test/test_all_grabber_types_generic.py

```diff
@@ -337,3 +337,21 @@ def test_determine_input_type_in_create(self):
         # this won't work because there are multiple input types that could be mapped to the serial_number field
         with self.assertRaises(ValueError):
             FrameGrabberConfig.create(serial_number="1234567890", resolution_width=640, resolution_height=480, digital_zoom=2)
+            
+    # def test_get_and_apply_options(self):
```

Comment:
Should we uncomment this test? I dont think we should leave it in if we're just going to keep it commented.